src/main/scala/nl.knaw.dans.easy.multideposit/actions/AddPropertiesToDeposit.scala

```diff
@@ -62,6 +62,7 @@ class AddPropertiesToDeposit extends DebugEnhancedLogging {
       "identifier.dans-doi.registered" -> Some("no"),
       "identifier.dans-doi.action" -> Some("create"),
       "bag-store.bag-name" ->  Some(bagDirName),
+      "deposit.source" ->  Some("SMD"),
```

Comment:
Following the comment in https://github.com/DANS-KNAW/easy-specs/pull/82#discussion_r310344792, please rename this property to `deposit.origin`. Also rename the parameter name and method name accordingly.